### PR TITLE
[TIP-435] Clean the css colors mess

### PR DESCRIPTION
**Pull request by pierallard**
tl;dr :

Set all the colors into the variables files, to override only this file when color change.

In details:

1) Replace the #xxxxxx hardcoded colors by variables
2) Put in the variables.less or bootstrap_variables.less
3) Remove the similar colors
4) Fix shit left and right
5) Change to have official header's color (part of TIP-5900 card)
6) Unify variable names: for each standard color (red, blue, purple, grey, darkblue, slategrey and akeneolight), you have dark, darker, light and lighter combinations. And that's all, we should not add any other. Particular one is grey, we have a superlight. Stephane will take a look how to simplify this after.

As you can see, there is no "drastic" changes: but we now have 18 greys instead of 90, 4 greens instead of 20, etc.
akeneo

### CODE REVIEW

**Comment by juliensnz** [comment]
GTM

**Comment by jjanvier** [comment]
GTM
